samples/hosting/multi-hosting/Version_5/Instance1/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NServiceBus" version="5.2.14" targetFramework="net45" />
+  <package id="NServiceBus" version="4.2.0" targetFramework="net45" />
```

Comment:
i think something is stuffed on git. perhaps rebase or do a clean get and resubmit PR